build.gradle

```diff
@@ -27,6 +26,7 @@ buildscript {
 }
 plugins {
     id "com.github.spotbugs"
+    id 'com.github.johnrengelman.shadow' version '7.0.0'
```

Comment:
Can we put it in `gradle/java.gradle` as before?

Reply:
fixed